### PR TITLE
adapt client code to new controller

### PR DESCRIPTION
**Pull request by reconeazaretzky**
adapted the client code to use the new base controller and service

### CODE REVIEW

File: src/main/java/com/entropyteam/entropay/employees/clients/controllers/ClientController.java [hunk 1]
```diff
@@ -1,78 +1,22 @@
...
-    @DeleteMapping(value = "/{clientId}")
-    public ResponseEntity<BodyBuilder> deleteClientById(@PathVariable String clientId) {
-        log.info("DELETE deleteClientById: {}.", clientId);
-        clientService.deleteClient(clientId);
-        return ResponseEntity.ok().build();
+    @Autowired
+    public ClientController(ClientService clientService) {
+        super(clientService);
     }
 }
```
Comment (agustinb4) [inline src/main/java/com/entropyteam/entropay/employees/clients/controllers/ClientController.java:22]: No newline at end of file

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -7,4 +7,11 @@ spring.jpa.properties.hibernate.default_schema=employees
...
-spring.datasource.password=entropay
\ No newline at end of file
+spring.datasource.password=entropay
+
+# setting these two properties based on best practice, for more info see https://vladmihalcea.com/connection-monitoring-jpa-hibernate/
+spring.datasource.hikari.auto-commit=false
+spring.jpa.properties.hibernate.connection.provider_disables_autocommit=true
+
+# disabling this property, for more info see https://stackoverflow.com/a/48222934/969252 and https://www.baeldung.com/spring-open-session-in-view
+spring.jpa.open-in-view=false
```
Comment (agustinb4) [inline src/main/resources/application.properties:17]: No newline at end of file

File: src/main/java/com/entropyteam/entropay/employees/clients/services/ClientService.java [hunk 3]
```diff
@@ -1,87 +1,77 @@
...
 
-        Pageable paging = PageRequest.of(--page, size).withSort(sort, sortBy);
-        Page<ClientDto> result = clientRepository.findAllByDeletedIsFalse(paging);
-        log.trace("Result: {}.", result);
-        return result;
+    @Override
+    @Transactional
+    public ClientDto update(UUID id, ClientDto entity) {
+        Client client = clientRepository.findById(id).orElseThrow();
+        client.setName(entity.name());
```
Comment (agustinb4) [inline src/main/java/com/entropyteam/entropay/employees/clients/services/ClientService.java:67]: could we leverage the mapper here?

**Review by agustinb4 — APPROVED** [review]
Looks fine

File: src/main/java/com/entropyteam/entropay/employees/clients/services/ClientService.java (hunk 3 above)
Comment (arael82) [reply src/main/java/com/entropyteam/entropay/employees/clients/services/ClientService.java:67]: yes, with clientMapper.toEntity :+1:

**Review by arael82 — APPROVED** [review]
Looks good to me.

**Pull request merged**